package.json

```diff
@@ -131,7 +131,7 @@
     "sinon": "^7.2.7",
     "style-loader": "^0.23.1",
     "stylelint": "^9.10.1",
-    "stylelint-config-idiomatic-order": "^6.2.0",
+    "stylelint-config-idiomatic-order": "^7.0.2",
```

Comment:
I tried to upgrade `stylelint` but there's an error in appveyor that I didn't want to investigate. So I only upgraded the plugin.